app/assets/stylesheets/application.scss

```diff
@@ -33,6 +33,21 @@ footer {
   padding-top: 15px;
 }
 
+.page-nav {
+  margin-top: 35px;
+  padding-left: 15px;
+  background-color: gray;
+  color: black;
+}
+
+.page-nav a {
+  color: black;
+}
+
+.page-nav a:active {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.